WORKSPACE

```diff
@@ -67,11 +60,19 @@ bind(
     actual = "//tools:java_toolchain",
 )
 
-load("@rules_java//java:repositories.bzl", "rules_java_dependencies", "rules_java_toolchains")
-
-rules_java_dependencies()
-
-rules_java_toolchains()
+"""
```

Comment:
please use `#` for comments. 

We only use `"""....."""` for comments in the first line in bazel files. All other comments use `#`.

Reply:
Done, thanks

Reply:
is it pushed? It seems like not done

Reply:
```
# The rules_proto contains the javalite_toolchain needed for the java_lite_proto_library rule used # in the model module.
```

Reply:
My bad, done and pushed now